test/provider/AppWindow.unittest.ts

```diff
@@ -4,15 +4,23 @@ import {Identity} from 'openfin/_v2/main';
 
 import {createMockChannel} from '../mocks';
 import {Application} from '../../src/client/main';
-import {AbstractAppWindow} from '../../src/provider/model/AppWindow';
+import {AppConnectionBase} from '../../src/provider/model/AppConnection';
 import {ContextChannel} from '../../src/provider/model/ContextChannel';
 import {useMockTime, unmockTime, advanceTime, resolvePromiseChain} from '../utils/unit/time';
+import {EntityType} from '../../src/provider/model/Environment';
 
-class TestAppWindow extends AbstractAppWindow {
+class TestAppWindow extends AppConnectionBase {
```

Comment:
Rename to `TestAppConnection`, or have it always pass `EntityType.WINDOW` to `super`?

Reply:
Went with the latter